editor/src/clj/editor/font.clj

```diff
@@ -404,14 +406,20 @@
         face-mask (if layer-mask-enabled
                     [1 0 0]
                     [1 1 1])
-        shadow-offset {:x (:shadow-x font-map), :y (:shadow-y font-map)}
+        shadow-offset {:x (if (:is-monospaced font-map)
+                                               (- (:shadow-x font-map) (* (:padding font-map) 0.5))
+                                               (:shadow-x font-map))
+                       :y (:shadow-y font-map)}
         alpha (:alpha font-map)
         outline-alpha (:outline-alpha font-map)
-        shadow-alpha (:shadow-alpha font-map)]
-    ;; Output glyphs per layer in back to front, if enabled but always output face layer.
+        shadow-alpha (:shadow-alpha font-map)
+        font-offset {:x (if (:is-monospaced font-map)
+                           (- 0 (* (:padding font-map) 0.5))
+                           0)
+                     :y 0}]
```

Comment:
We render each glyph starting some point and then draw every next letter at Advance Width (+ tracking) from prev. one.

But width of the glyph already include padding. Which makes width of the text a bit bigger and moves initial rendering point for the padding. 

Adding padding the line-widths helps to return real line width and compensate 0.5 of padding for the positioning . But another 0.5 of padding should be compensted by moving the initial point of rendering (we can't just add double padding to the line-width)